py4DSTEM/process/diskdetection/diskdetection.py

```diff
@@ -343,8 +364,16 @@ def find_Bragg_disks_serial(datacube, probe,
     t = time()-t0
     print("Analyzed {} diffraction patterns in {}h {}m {}s".format(datacube.R_N, int(t/3600),
                                                                    int(t/60), int(t%60)))
-
-    return peaks
+    
+    if global_threshold == True:
+        peaks = universal_threshold(peaks, minGlobalIntensity, metric, minPeakSpacing, maxNumPeaks)
+        
+    if not hist:
```

Comment:
Did you want me to remove the universal threshold as well, or just  the histogram functionality? I'm fine with either. My workflow looked like the following:

find_bragg_disks_serial()
get_pointlistarray_hist()
universal_threshold()

However if people are working on several datasets from the same dat it might be helpful to just apply the same universal threshold during disk detection. I'm fine with whether you want the UT incorporated into the serial detection or not. I don't know if it will make a huge difference to users.

Reply:
Hmm.  I'm not certain...but I was thinking remove it here, so that users are required to use the workflow you describe above.  It might actually be quite helpful to break it down even a little more, such that the user can look at the results of setting some threshold before actually applying it.  I would imagine something like tagging which peaks will be removed without actually removing them, combined with a vis function that displays only the untagged peaks, or something like this.  But this is really a suggestion for a later feature, I don't want to hold this up now.